sql/moz-fx-data-shared-prod/org_mozilla_fenix/releases/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.org_mozilla_fenix.releases`
+AS
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.org_mozilla_fenix_derived.releases_v1`
```

Comment:
Can we also provide a metadata.yaml for each view to make sure they're included in generated docs?